src/main/java/dibo/TaskList.java

```diff
@@ -0,0 +1,144 @@
+package dibo;
+
+import java.util.ArrayList;
+
+import dibo.exception.DiboException;
+import dibo.task.Task;
+
+/**
+ * Class to store all the tasks.
+ */
+public class TaskList {
+    private final ArrayList<Task> tasks;
+    private int count;
+
+    /**
+     * Constructs the TaskList class.
+     *
+     * @param storage The ArrayList returned from loading from the text file.
+     */
+    public TaskList(ArrayList<Task> storage) {
+        this.tasks = storage;
+        this.count = storage.size();
+    }
+
+    /**
+     * Takes in a task and adds it into the TaskList.
+     *
+     * @param task The task to be added.
+     */
+    public void addTask(Task task) {
+        this.tasks.add(task);
+        this.count++;
+    }
+
+    /**
+     * Returns the string representation of the list to be displayed.
+     */
+    public String getDisplayFormat() {
+        StringBuilder list = new StringBuilder("Here are the tasks in your list:\n");
+        for (int i = 0; i < count; ++i) {
+            list.append((i + 1));
+            list.append(".");
+            list.append(tasks.get(i).toString());
+            list.append("\n");
```

Comment:
Perhaps you can consider using `String.format()` to have it in 1 line?

Reply:
Thank you for the suggestion! I will look into it!